apps/studio/components/interfaces/Support/SupportFormV2.tsx

```diff
@@ -338,6 +354,9 @@ export const SupportFormV2 = ({ setSentCategory, setSelectedProject }: SupportFo
                       {organizations?.map((org) => (
                         <SelectItem_Shadcn_ value={org.slug}>{org.name}</SelectItem_Shadcn_>
                       ))}
+                      {organizations?.length === 0 && (
```

Comment:
just to check - do we want this option only available if the user has 0 organizations? or make it available for everyone?

i'm okay with either tbh - the current implementation will not have any change for most users i guess which is also good

Reply:
> do we want this option only available if the user has 0 organizations? 

this, see PR description for reasining

Reply:
ahhh gotcha! sorry skipped over that 🙏  gotcha on this yup thats all fine